### PR TITLE
Fixing up more classes for #88

### PR DESCRIPTION
**Pull request by gtfierro**
- adds (average) heating/cooling demand sensor
- standardize tags to `heat`/`cool` rather than `heating`/`cooling`

### CODE REVIEW

**Comment by gtfierro** [comment]
@shreyasnagare let me know if there are any more outstanding classes left for #88 and I can add them in this PR

File: bricksrc/sensor.py [hunk 1]
```diff
@@ -577,6 +595,21 @@
                             },
                         },
                     },
+                    "Energy_Usage_Sensor": {
+                        "parents": [BRICK.Energy_Sensor],
+                        "tags": [TAG.Point, TAG.Sensor, TAG.Energy, TAG.Usage],
+                        "subclasses": {
+                            "Yearly_Usage_Energy_Sensor": {
```
Comment (jbkoh) [inline bricksrc/sensor.py]: The order of tags in the class names change in these subclasses from `Energy_Usage_Sensor` to `Usage_Energy_Sensor`. I prefer the former one though I have no strong opinion.

Comment (gtfierro) [reply bricksrc/sensor.py]: Sorry; I'm very confused by what you mean there. Can you rephrase? Are you saying the order of words in `"tags"` is inconsistent? Or the class names are inconsistent?

Comment (jbkoh) [reply bricksrc/sensor.py]: I meant the class names. I want to change `Yearly_Usage_Energy_Sensor` to `Yearly_Energy_Usage_Sensor` (same for its siblings). Their superclass's name is `Energy_Usage_Sensor`.

Comment (gtfierro) [reply bricksrc/sensor.py]: Ah, makes sense. Good change

**Comment by gtfierro** [comment]
@shreyasnagare @jbkoh am I good to merge this now?

**Comment by shreyasnagare** [comment]
@gtfierro Looks good to me.

**Comment by jbkoh** [comment]
LGTM.

**Comment by gtfierro** [comment]
Thanks for all your diligent work on this @shreyasnagare and @jbkoh . Really appreciate the thorough feedback :)
